mass_editing.py

```diff
@@ -164,8 +164,11 @@ def _object_hook(self, dct):
                 if company_field:
                     model_fields = model_fields + (company_field[0],)
 
-        fields.update(EditingModel.fields_get([f.name for f in
-                    model_fields]))
+        model_field_names = [f.name for f in model_fields]
+        for k, v in EditingModel.fields_get(model_field_names).items():
+            # sure field_name key from fields_get is request in edit.model_fields
```

Comment:
sure -> Ensure
is request -> is requested

Reply:
cb11df1